### PR TITLE
[#76] Document baking on Raspberry Pi OS

### LINKED ISSUE #76: Provide .deb packages for Raspbery Pi OS
# Clarification and motivation

There is Raspbery Pi OS for Raspbery Pi. It supports .deb packages. As result of #70 we can now build static binaries for arm64, but some people may prefer native .deb packages.

# Acceptance criteria

Our CI produces .deb packages that can be uploaded to the common place where they are uploaded for arm64, so that one can do something similar to
```
sudo add-apt-repository ppa:serokell/tezos && sudo apt-get update
sudo apt-get install tezos-client
```

on their Raspbery Pi running Raspbery Pi OS.

If #63 is done before this issue, we would like to also upload them automatically from CI.

### PR DESCRIPTION
**Pull request by pasqu4le**
## Description

Updates the README and baking tutorial with installation info for Raspberry Pi OS.

## Related issue(s)

Resolves #76

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../tree/master/README.md)

- [x] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,26 @@ sudo apt-get install tezos-baker-007-psdelph1
 ```
 Once you install such packages the commands `tezos-*` will be available.
 
+### Ubuntu packages on Raspberry Pi OS
```
Comment (rvem) [inline README.md:66]: Let's add this paragraph to the table of contents at the begging of the README

**Review by rvem — APPROVED** [review]
LGTM, have only one suggestion

**Comment by pasqu4le** [comment]
Note: removed Draft as there are no problems with the latest OS image and native Ubuntu packages.

File: README.md [hunk 2]
```diff
@@ -63,6 +64,27 @@ sudo apt-get install tezos-baker-007-psdelph1
...
+If you have a Raspberry Pi running the 64bit version of the official OS, you can
+use the Lauchpad PPA to install `tezos-*` executables on it as well.
+
+You can add the PPA using:
+```
+# Intall software properties commons
+sudo apt-get install software-properties-common
+# Add PPA with Tezos binaries
+sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bionic main'
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com --recv-keys 37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB
```
Comment (itkach) [inline README.md:79]: this doesn't appear to be a valid key: command fails and [lookup via web interface](https://keyserver.ubuntu.com/pks/lookup?search=37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB&fingerprint=on&op=index) also returns "not found"

Comment (pasqu4le) [reply README.md:79]: @itkach I wonder if it was a momentary failure, could you try again?
I checked and the key seems to be correct.

You can see that's the signing key from the [lauchpad page](https://launchpad.net/~serokell/+archive/ubuntu/tezos) (under the `Technical details about this PPA`).

The ubuntu keyserver web interface seems to require the `0x` prefix, with that the [key can be found](https://keyserver.ubuntu.com/pks/lookup?search=0x37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB&fingerprint=on&op=index).

I also checked again on my Pi and was able to `apt-get update` and `upgrade` to `v.9`, as well as delete and re-import the key with the command above.

Comment (itkach) [reply README.md:79]: advice here helped: https://www.laurivan.com/apt-key-adv-fix-server-indicated-a-failure-on-raspberry-pi/ - key was successfully imported after changing keyserver url to `hkp://keyserver.ubuntu.com:80` (specify port 80 explicitly)

Comment (pasqu4le) [reply README.md]: :ok_hand: added the port explicitly now. Oddly both with and without it seem to work for me.

**Pull request merged**